package.json

```
@@ -43,25 +43,25 @@
     "@testing-library/dom": "^9.3.3",
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "^14.0.0",
-    "@trivago/prettier-plugin-sort-imports": "^4.2.0",
+    "@trivago/prettier-plugin-sort-imports": "^4.2.1",
     "@types/jest": "^29.5.6",
     "@types/jest-axe": "^3.5.7",
-    "@types/node": "^20.8.7",
-    "@types/react": "^18.2.28",
+    "@types/node": "^20.8.8",
+    "@types/react": "^18.2.31",
     "autoprefixer": "^10.4.16",
     "axe-core": "^4.8.2",
     "babel-jest": "^29.7.0",
-    "cypress": "^13.3.1",
+    "cypress": "^13.3.2",
     "cypress-axe": "^1.5.0",
     "cypress-multi-reporters": "^1.6.3",
-    "eslint": "8.51.0",
-    "eslint-config-next": "^13.5.5",
+    "eslint": "8.52.0",
```

```suggestion
    "eslint": "^8.52.0",
```

(to match all the others)

---

Fixed